Piwik.Tracker/PiwikTracker.cs

```diff
@@ -585,10 +606,10 @@ protected string GetCookieName(string cookieName)
         /// </summary>
         /// <param name="documentTitle">Page title as it will appear in the Actions > Page titles report</param>
         /// <returns>HTTP Response from the server or null if using bulk requests.</returns>
-        public TrackingResponse DoTrackPageView(string documentTitle = null)
+        public Task<TrackingResponse> TrackPageViewAsync(string documentTitle = null)

```

Comment:
[You should add “Async” as the suffix of every async method name you write.](https://docs.microsoft.com/en-us/dotnet/articles/csharp/async)